shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -70,6 +72,36 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
   return NO;
 }
 
+- (void)applyBackdropFilter {
+  if (_backdropFilterApplied)  // TODO EMILY: This check is not needed, but it might improve
+                               // space/time performance.
+    return;
+
+  UIVisualEffect* blurEffect;
+  blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleLight];
+
+  UIVisualEffectView* visualEffectView;
+  visualEffectView = [[UIVisualEffectView alloc] initWithEffect:blurEffect];
+
+  UIView* view = [visualEffectView.subviews firstObject];
+  [visualEffectView release];
+  if (!view)  // TODO EMILY: Do we also need to check that the view received from UIVEV is the
+              // expected  _UIVisualEffectSubview? The next check will catch discrepencies, but we
+              // could return earlier here.
+    return;
+
+  NSObject* gaussianFilter = [view.layer.filters firstObject];
+  if (!gaussianFilter || ![[gaussianFilter valueForKey:@"name"] isEqual:@"gaussianBlur"])
+    return;
+
+  [gaussianFilter setValue:@(5)
```

Comment:
Try to see if [validateValue:forkey:error](https://developer.apple.com/documentation/objectivec/nsobject/1416754-validatevalue?language=objc) works to avoid crashing on production code.